internal-webapps/org.wso2.bfsi.authentication.endpoint/src/main/webapp/privacy_policy.jsp

```diff
@@ -0,0 +1,238 @@
+                    <p>WSO2 Open Banking stores your personal information in secured databases. WSO2 Open Banking exercises proper industry accepted security measures to protect the database where your personal information is held. WSO2 Open Banking as a product does not transfer or share your data with any third parties or locations. </p>
+                    <p>WSO2 Open Banking may use encryption to keep your personal data with an added level of security.</p>
+                </section>
+                
+                <section style="margin-bottom: 2em;">
+                    <h3 id="how-long-does-is-5.5-keep-your-personal-information" style="font-weight:400; color:black">How long your personal information is retained</h3>
+                    <p>WSO2 Open Banking retains your personal data as long as you are an active user of our system. You can update your personal data at any time using the given self-care user portals.</p>
+                    <p>WSO2 Open Banking may keep hashed secrets to provide you with an added level of security. This includes:</p>
+                    <ul>
+                        <li>Current password</li>
+                        <li>Previously used passwords</li>
+                    </ul>
+                </section>
+                
+                <section style="margin-bottom: 2em;">
+                    <h3 id="how-to-request-removal-of-your-personal-information" style="font-weight:400; color:black">How to request removal of your personal information</h3>
+                    <p>You can request the administrator to delete your account. The administrator is the administrator of the tenant you are registered under, or the super-administrator if you do not use the tenant feature.</p>
+                    <p>Additionally, you can request to anonymize all traces of your activities that WSO2 Open Banking may have retained in logs, databases or analytical storage.</p>
+                </section>
+                
+                <section style="margin-bottom: 2em;">
+                    <h2 id="more-information" style="font-weight:400; color:black">More information</h2>
+                    <div class="margin-bottom"></div>
+                </section>
+                
+                <section style="margin-bottom: 2em;">
+                    <h3 id="changes-to-this-policy" style="font-weight:400; color:black">Changes to this policy</h3>
+                    <p>Upgraded versions of WSO2 Open Banking may contain changes to this policy and revisions to this policy will be packaged within such upgrades. Such changes would only apply to users who choose to use upgraded versions.</p>
+                    <p>The organization running WSO2 Open Banking may revise the Privacy Policy from time to time. You can find the most recent governing policy with the respective link provided by the organization running WSO2 IS 5.5. The organization will notify any changes to the privacy policy over our oficial public channels.</p>
```

Comment:
```suggestion
                    <p>The organization running WSO2 Open Banking may revise the Privacy Policy from time to time. You can find the most recent governing policy with the respective link provided by the organization running WSO2 IS 7.0 The organization will notify any changes to the privacy policy over our oficial public channels.</p>
```

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/8/commits/55101612f0aa99105f83a6d7d3d648000eb13112